findspam.py

```diff
@@ -342,14 +342,15 @@ def has_health(s, site, *args):   # flexible detection of health spam in titles
 
 # noinspection PyUnusedLocal,PyMissingTypeHints
 def pattern_product_name(s, site, *args):
-    keywords = ["Testo?", "Dermapholia", "Garcinia", "Cambogia", "Aurora", "Kamasutra", "HL-?12", "NeuroFuse",
+    keywords = ["Testo?(?:sterone)?", "Dermapholia", "Garcinia", "Cambogia", "Aurora", "Diet", "Slim", "Premier",
```

Comment:
Note that testosterone was deliberately not caught when I originally designed this rule